boilerplate/app/i18n/jp.ts

```diff
@@ -0,0 +1,127 @@
+import { Translations } from "./en"
```

Comment:
@benjaminkomen the objective here is to create type-safety amongst the different language files. For us, `en` is the default language and we want to make sure all other files have all the keys, since they're required.

This is what is imported on that line: `export type Translations = typeof en`

If there is another way of accomplishing this, please let us know.